index.html

```diff
@@ -224,8 +224,20 @@ <h2 id="faq">Frequently Asked Questions <a href="#faq"><svg width="24" height="2
         <p class="font-bold" id="regular-user">
           I'm a regular Terraform user, and I'm not competing with HashiCorp. Why should I care? <a href="#regular-user"><svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"> <path d="M13.5442 10.4558C11.8385 8.75022 9.07316 8.75022 7.36753 10.4558L4.27922 13.5442C2.57359 15.2498 2.57359 18.0152 4.27922 19.7208C5.98485 21.4264 8.75021 21.4264 10.4558 19.7208L12 18.1766" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/> <path d="M10.4558 13.5442C12.1614 15.2498 14.9268 15.2498 16.6324 13.5442L19.7207 10.4558C21.4264 8.75021 21.4264 5.98485 19.7207 4.27922C18.0151 2.57359 15.2497 2.57359 13.5441 4.27922L12 5.82338" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/> </svg></a>
         </p>
+
+        <p>
+          There are two reasons that you, as a regular Terraform user, should
+          care.
+        </p>
+        <p>
+          First, if you want to automate your Terraform process, like you would
+          with any other code, you are limited to either spending your precious
+          engineering time and money to build your own solution or using
+          Terraform Cloud.  HashiCorp is removing options from you in how best
+          to automate your Terraform process.
+        </p>
         <p>
-          How do you know you're not competing with HashiCorp? 
+          Secondly, how do you know you're not competing with HashiCorp? 
```

Comment:
I guess the third argument is that a license change is a slippery slope. Done once it can be done again if the 1st decision does not yield expected business results. Say you require payment for any production use (so, TF remains free, but for non-production use only).

In that sense what should worry everyday users is not the current license, but the fact that the license owner is desperately trying to squeeze out more value from it.

Even if you don't care for OpenTF, and want to continue using Terraform under BSL, your best bet is to support OpenTF to remove the incentive for Hashi to further limit your options.

Hope that makes sense.